### DISCUSSION TITLE
How to check the version of OceanParcels?

### DISCUSSION THREAD

**Discussion by VincentIMEDEA** [Parcels-code/Parcels#1468] [General]
Hi,

I could not find this info online.
What is the command to check the version of Parcels that I use?

Thank you

Vincent

**Comment by erikvansebille** [comment]
Hi Vincent

There are two easy ways:
1. `import parcels` and then `print(parcels.__version__)`
2. Look into the metadata of the `zarr` file created, which has a `parcels_version` entry
